src/edu/wright/cs/raiderplanner/model/Exam.java

```diff
@@ -58,13 +58,15 @@ public ExamEvent getTimeSlot() {
 
 
 	// constructors
-	public Exam(int cweighting, Person csetBy, Person cmarkedBy, Person creviewedBy, int cmarks, ExamEvent ctimeSlot, Exam cresit) {
+	public Exam(int cweighting, Person csetBy, Person cmarkedBy, Person creviewedBy, 
+						int cmarks, ExamEvent ctimeSlot, Exam cresit) {
 		super(cweighting, csetBy, cmarkedBy, creviewedBy, cmarks);
 		timeSlot = ctimeSlot;
 		resit = cresit;
 	}
 
-	public Exam(int cweighting, Person csetBy, Person cmarkedBy, Person creviewedBy, int cmarks, ExamEvent ctimeSlot) {
+	public Exam(int cweighting, Person csetBy, Person cmarkedBy, Person creviewedBy,
+						int cmarks, ExamEvent ctimeSlot) {
```

Comment:
The continuation line should only be indented two additional tabs from the line above.